src/test/java/marquez/db/DatasetDaoTest.java

```diff
@@ -0,0 +1,54 @@
+package marquez.db;
+
+import static org.junit.Assert.assertEquals;
+
+import marquez.common.models.Namespace;
+import marquez.core.models.Generator;
+import marquez.db.models.DataSourceRow;
+import marquez.db.models.DatasetRow;
+import marquez.db.models.DbTableInfoRow;
+import marquez.db.models.DbTableVersionRow;
+import org.jdbi.v3.core.Jdbi;
+import org.jdbi.v3.sqlobject.SqlObjectPlugin;
+import org.jdbi.v3.testing.JdbiRule;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class DatasetDaoTest {
+  static final Logger logger = LoggerFactory.getLogger(DatasetDaoTest.class);
+  private DatasetDao datasetDAO;
+  private marquez.core.models.Namespace namespace = Generator.genNamespace();
+  private NamespaceDao namespaceDAO;
+
+  @Rule
+  public final JdbiRule dbRule =
+      JdbiRule.embeddedPostgres().withPlugin(new SqlObjectPlugin()).migrateWithFlyway();
+
+  @Before
```

Comment:
Can we set these on a per-class basis instead of a per-test-method instance? If so, we can use `@BeforeClass` instead of `@Before`.